yarn.lock

```diff
@@ -4083,6 +4083,16 @@ browserslist@^4.0.0, browserslist@^4.14.5, browserslist@^4.16.6, browserslist@^4
     node-releases "^2.0.6"
     update-browserslist-db "^1.0.9"
 
+browserslist@^4.18.1:
```

Comment:
What's this doing in here 👀

Reply:
May want to [`yarn-deduplicate`](https://www.npmjs.com/package/yarn-deduplicate)

Reply:
Done now.